mlflow/transformers/model_io.py

```diff
@@ -145,6 +145,13 @@ def _load_model(model_name_or_path, flavor_conf, accelerate_conf, device, revisi
     if torch_dtype := flavor_conf.get(FlavorKey.TORCH_DTYPE):
         load_kwargs[FlavorKey.TORCH_DTYPE] = torch_dtype
 
+    if torch_dtype := flavor_conf.get(FlavorKey.TORCH_DTYPE):
+        import torch
+
+        attr = torch_dtype.rsplit(".", 1)[-1]
```

Comment:
Assuming `torch_dtype` is a string that looks like `torch.xxx`.

Reply:
Can we use https://github.com/mlflow/mlflow/blob/master/mlflow/transformers/torch_utils.py#L42?

Reply:
nice, we can!

Reply:
We have tests for torch dtype. Wondering why they don't hit this error.

Reply:
We have steps to uninstall accelerate and run tests again, but only 2 test cases are executed there... https://github.com/mlflow/mlflow/blob/master/mlflow/ml-package-versions.yml#L552-L553

I think we should run more and there are a few options;
1. Simply running all tests after uninstalling accelerate. But this will be very heavy load.
2. Add a few (~10) important tests in `-k` flag (or via annotation). But the criteria for adding/not-adding is not super clear.
3. Run all tests just for the latest version of transformers. Basically add one more run to test matrix that `latest x no-accelerate`. This way we can at least ensure that the all test cases work with no-accelerate env, without increasing computation cost too much.

What do you think? @harupy

Reply:
Created a follow-up in the next sprint (ML-39046)